src/lib/index.js

```diff
@@ -1,61 +1,61 @@
 // Layout components
-export { default as Center } from './components/layout/Center';
+export { default as Center } from './components/Center';
 export {
   CTA,
   CTACenter,
   CTAEnd,
   CTAStart,
-} from './components/layout/CTA';
+} from './components/CTA';
 export {
   FormLayout,
   FormLayoutCustomField,
-} from './components/layout/FormLayout';
+} from './components/FormLayout';
 export {
   Grid,
   GridSpan,
-} from './components/layout/Grid';
+} from './components/Grid';
 export {
   List,
   ListItem,
-} from './components/layout/List';
+} from './components/List';
 export {
   Media,
   MediaBody,
   MediaObject,
-} from './components/layout/Media';
+} from './components/Media';
 export {
   Toolbar,
   ToolbarGroup,
   ToolbarItem,
-} from './components/layout/Toolbar';
+} from './components/Toolbar';
 
 // UI components
```

Comment:
Let's get rid of these comments and flatten the exports in this file as well.

Reply:
done